radix-transactions/src/model/v1/test_transaction.rs

```diff
@@ -45,6 +120,39 @@ pub struct PreparedTestIntent {
     pub initial_proofs: BTreeSet<NonFungibleGlobalId>,
 }
 
+impl PreparedTestIntent {
+    #[allow(deprecated)]
```

Comment:
Should be 
```suggestion
    #[deprecated]
```

Reply:
Building a V1 test intent is fine, because it's a legacy but supported standard. So I don't view it as being deprecated.

Preparing of instructions in a v1 way is deprecated though because it uses an old, less good, model for what to hash. But obviously it's ok (and required) to use the deprecated method for V1 transactions. We have to add an `#[allow(deprecated)]` here to avoid a compiler warning.

Reply:
Fair enough :)